src/webview/leetCodeSubmissionProvider.ts

```diff
@@ -40,6 +42,15 @@ class LeetCodeSubmissionProvider extends LeetCodeWebview {
         super.onDidDisposeWebview();
         delete this.result;
     }
+
+    private async showKeybindingsHint(): Promise<void> {
+        await promptHintMessage(
+            "hint.commandShortcut",
+            'You can configure custom key bindings with "test", "submit" or any other command in Preferences > Keyboard Shortcuts.',
```

Comment:
You can customize the shortcut key bindings in Preferences > Keyboard Shortcuts.